libraries/AP_HAL_Linux/Util.h

```diff
@@ -48,7 +48,7 @@ class Util : public AP_HAL::Util {
      */
     void set_system_clock(uint64_t time_utc_usec);
     const char* get_custom_log_directory() { return custom_log_directory; }
-    const char* get_custom_terrain_directory() { return custom_terrain_directory; }
+    const char* get_custom_terrain_directory() const final { return custom_terrain_directory; }
```

Comment:
There is no such thing as an overriding method that isn't virtual, if that was the case then final would have no use, you would just take out the virtual and the method couldn't be overridden.

We should add `override` in every method when we are doing these changes, it prevents bugs like this from happening.